sonarts-core/src/utils/parser.ts

```diff
@@ -69,7 +69,9 @@ export function parseFile(filename: string): { sourceFile: ts.SourceFile; progra
 export function createProgram(configFile: string): ts.Program {
   const { options, files } = parseTsConfig(configFile);
   const host = ts.createCompilerHost(options, true);
-  return ts.createProgram(files, options, host);
+  // make sure traceResolution is disabled because it generates a lot of content on stdout
+  const optionsWithoutTrace = Object.assign(options, { traceResolution: false });
```

Comment:
Don't you have a way to unit test directly this?

The IT is pretty cool and a nice addition (which I managed to use correctly... bug was on the other side of the keyboard), but I wonder if it wouldn't be also good to have a dedicated unit test just checking that we are never calling createProgram() with options having `traceResolution` being true.

Reply:
I agree: I think I will move those two added lines inside `parseTsConfig` function and will unit test it. Thanks!